pkgs/development/python-modules/cnvkit/default.nix

```diff
@@ -21,22 +21,23 @@
 
 buildPythonPackage rec {
   pname = "CNVkit";
-  version = "0.9.7";
+  version = "0.9.9";
 
   src = fetchFromGitHub {
     owner = "etal";
     repo = "cnvkit";
     rev = "v${version}";
-    sha256 = "022zplgqil5l76vri647cyjx427vnbg5r2gw6lw712d2janvdjm7";
+    sha256 = "1q4l7jhr1k135an3n9aa9wsid5lk6fwxb0hcldrr6v6y76zi4gj1";
   };
 
-  patches = [
-    # Fix: AttributeError: module 'pandas.io.common' has no attribute 'EmptyDataError'
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
MB, shouldn't have missed that.